spread.yaml

```diff
@@ -11,6 +11,34 @@ exclude:
   - .github
 
 backends:
+  lxd:
```

Comment:
I noticed this comment missing here, compared to 24.04:

```diff
-  # We can switch back to using the official Spread LXD backend after
-  # https://github.com/snapcore/spread/pulls?q=is%3Apr+is%3Aopen+LXD+updated%3A%3C%3D2024-02-23+
```

Does this not apply here?

Reply:
added